modules/core/src/precomp.hpp

```diff
@@ -258,15 +258,13 @@ struct ImplCollector
 struct CoreTLSData
 {
     CoreTLSData() :
-//#ifdef HAVE_OPENCL
         device(0), useOpenCL(-1),
-//#endif
```

Comment:
@alalek, could you provide more details why these lines should be left?
(they look like a garbage)